README.md

```diff
@@ -9,27 +9,59 @@
 
 This menu library is made to easily create an inline menu for your Telegram bot.
 
-## Example
-
 ![Example Food Menu](media/example-food.gif)
+
+# Installation
+
+```
+$ npm install telegraf-inline-menu
+```
+
+or using `yarn`:
+
+```
+$ yarn add telegraf-inline-menu
+```
+
+# Examples
+
+## Basic Example
+
 ```js
-const menu = new TelegrafInlineMenu(
-  ctx => `Hey ${ctx.from.first_name}!`
-)
+const Telegraf = require('telegraf')
+const TelegrafInlineMenu = require('telegraf-inline-menu')
+
+const menu = new TelegrafInlineMenu(ctx => `Hey ${ctx.from.first_name}!`)
+
 menu.simpleButton('I am excited!', 'a', {
   doFunc: ctx => ctx.reply('As am I!')
 })
+
+menu.setCommand('start')
+
+const bot = new Telegraf(process.env.BOT_TOKEN)
+
 bot.use(menu.init())
+
+bot.startPolling()
 ```
 
+## More interesting one
+
+Look at the code here : [example.js](example.js)
```

Comment:
Add the TypeScript one too as the library itself is written in TypeScript. Might be helpful to someone.